src/app/static/src/scss/partials/stepper.scss

```diff
@@ -16,10 +16,17 @@
     width: 40px;
   }
 
+  .btn:disabled {
+    cursor: default;
+  }
+
+  .btn:enabled {
```

Comment:
I wanted to show visually when the second step was disabled, and I'd wondered if the existing style, with grey border, looked a bit too disabled anyway. So see what you think of this - enabled but non-active gets a narrow mint border, disabled gets the grey border and default cursor.

Reply:
I like the addition of the green border. The difference between enabled and disabled is still quite subtle, how about also greying out the step text when disabled?